pkg/swarm/swarm.go

```diff
@@ -179,10 +190,18 @@ func (c *chunk) TagID() uint32 {
 	return c.tagID
 }
 
-func (c *chunk) Stamp() []byte {
+func (c *chunk) Stamp() Stamp {
 	return c.stamp
 }
 
+func (c *chunk) Radius() uint8 {
```

Comment:
comments would be ideal.

     Radius is the PO above which the batch is preserved.